lib/port.js

```diff
@@ -91,23 +91,25 @@ Port.prototype.unlink = function(err, cb) {
     }
 
     if (err) {
-        // unknown error
-        cb(err);
+        cb(LusterPortError
+            .createError(LusterPortError.CODES.UNKNOWN_ERROR, err));
```

Comment:
Yeah, but it's better than

~~~js
LusterPortError.createError(undefined, err);
~~~

Isn't it?

https://github.com/nodules/terror/blob/88c6541d5fedbc4bf93aa17a79bcc2387db16415/lib/terror.js#L164-L178